File: developer/sdk/java/AcosClient.java
package io.acos.sdk;

public class AcosClient {
    private String apiKey;
    private String baseUrl;

    private AcosClient(String apiKey, String baseUrl) {
        this.apiKey = apiKey;
        this.baseUrl = baseUrl;
    }

    public static AcosClient init(String apiKey) {
        return new AcosClient(apiKey, "https://api.acos.io/v1");
    }
}

File: developer/sdk/dotnet/AcosClient.cs
namespace Acos.Sdk
{
    public class AcosClient
    {
        public string ApiKey { get; }
        public string BaseUrl { get; }

        private AcosClient(string apiKey, string baseUrl)
        {
            ApiKey = apiKey;
            BaseUrl = baseUrl;
        }

        public static AcosClient Init(string apiKey)
        {
            return new AcosClient(apiKey, "https://api.acos.io/v1");
        }
    }
}
